src/Services/Masa.Auth.Service.Admin/appsettings.Development.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=10.10.90.37,30110;Database=masa-auth-v2;User Id=sa;Password=p@ssw0rd;"
```

Comment:
10.10.90.37?  Can we use the virtual domain?

Reply:
waiting domain name